docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,102 @@
+# Release 381 (13 May 2022)
+
+## General
+
+* Add support for table functions. ({issue}`1839`, {issue}`11937`)
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage . ({issue}`12211`)
+* Increase the default number of stages allowed for a query from 100 to 150,
+  specified with `query.max-stage-count`, so large queries do not fail. ({issue}`12292`)
+* Use the `fault_tolerant_execution_partition_count` session property, or the
+  `fault-tolerant-execution-partition-count` configuration property, to adjust
+  the number of partitions for distributed joins and aggregations executed with
+  fault-tolerant execution enabled (`retry-policy=TASK`). ({issue}`12263`)
+* Rename configuration property used for selecting a policy for low memory query
+  killer from `query.low-memory-killer.policy` to
+  `query.low-memory-query-killer.policy`. ({issue}`12242`)
+* Introduce specific configuration property
```

Comment:
The property should be named `task.low-memory-killer.policy`, and the query-level memory killer property should stay as is. cc @losipiuk

Reply:
This item should be folded into the next one:

> Introduce `least-wasted` low memory task killer policy. This policy avoids xxx. It can be enabled via the yyy configuration propery

Reply:
> The property should be named `task.low-memory-killer.policy`, and the query-level memory killer property should stay as is. cc @losipiuk

I agree it would read nicer. The downside `query.low-memory-killer.delay` (which is effective also for task killing) has different prefix then. On the other hand we are suggesting to 
set `query.low-memory-killer.delay` to 0 if you have task level retries enabled, so it would make sense to ignore the delay property at all for task killing. I will do the rename.

Reply:
https://github.com/trinodb/trino/pull/12393

Reply:
@mosabua I did renames as suggested. We can drop the
```
Rename configuration property used for selecting a policy for low memory query
  killer from `query.low-memory-killer.policy` to
  `query.low-memory-query-killer.policy`. ({issue}`12242`)
```
And change `query.low-memory-task-killer.policy` to `task.low-memory-killer.policy` in following section